File: Program's_Contributed_By_Contributors/Java_Programs/caesar cipher.java
import java.util.Scanner;

class CaesarCipher
{
    public static StringBuffer encrypt(String text, int s)
    {
        StringBuffer result= new StringBuffer();
 
        for (int i=0; i<text.length(); i++)
        {
            if (Character.isUpperCase(text.charAt(i)))
            {
                char ch = (char)(((int)text.charAt(i) +
                                  s - 65) % 26 + 65);
                result.append(ch);
            }
            else
            {
                char ch = (char)(((int)text.charAt(i) +
                                  s - 97) % 26 + 97);
                result.append(ch);
            }
        }
        return result;
    }
 
    public static void main(String[] args)
    {
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter Plain Test: ");  
        String text = sc.nextLine();
        System.out.print("Enter Key: ");  
        int s = sc.nextInt(); 
        System.out.println("\n");
        System.out.println("Text  : " + text);
        System.out.println("Shift : " + s);
        System.out.println("Cipher: " + encrypt(text, s));
    }
}

File: Program's_Contributed_By_Contributors/Python_Programs/caesar_cipher.py
def encrypt(text,s):
	result = ""
	for i in range(len(text)):
		char = text[i]
		if (char.isupper()):
			result += chr((ord(char) + s-65) % 26 + 65)
		else:
			result += chr((ord(char) + s - 97) % 26 + 97)

	return result

text = "LEETCODE"
s = 4
print ("Your Text is : " + text)
print ("Shift : " + str(s))
print ("Cipher : " + encrypt(text,s))
